### DISCUSSION TITLE
Off-colors and weird triangle for the live view from certain cameras after updating to 0.14?

### DISCUSSION THREAD

**Discussion by zestysoft** [blakeblackshear/frigate#14417] [Ask A Question] [answered]
I've attached a quick screen grab of the problem -- if I switch to another area of the program (say to review alerts) and switch back, the video starts out looking fine, but then changes to what you'll see.  This happens for two cameras.  Fwiw, this is new behavior that did not show with the .13 birdseye view.

Both cameras have a 1920x536 resolution and use rtsp for record and detect. (Annke NCD800 4k)

https://github.com/user-attachments/assets/a4cda012-01fa-490f-a358-2385c90d64ec

**Comment by NickM-27** [comment]
See the FAQ usage question 15 https://github.com/blakeblackshear/frigate/discussions/12724

**Reply by zestysoft** [reply]
Thanks Nick!  I swear I read through a faq list, but apparently not the right one.  Switching to go2rtc solved it for me.

**Reply by zestysoft** [reply]
Actually that didn't solve it.

Here's the text from the FAQ:
```
This is caused by incorrect dimensions set in your detect width or height (or incorrectly auto-detected) that don’t match your camera, causing the jsmpeg player's rendering engine to display a slightly distorted image. See https://github.com/blakeblackshear/frigate/discussions/12700, https://github.com/blakeblackshear/frigate/discussions/13244 and https://github.com/blakeblackshear/frigate/discussions/14300. You should specify the correct dimensions in your config file. However, if you've done this and are still having issues, some users have reported success with ensuring that either of the dimensions (width or height) are numbers that are divisible by 4 (example: 800x443 becomes 800x444). Other users have also reported success by enlarging the width and height up to a standard aspect ratio (example: 640x352 becomes 640x360, a standard 16:9 aspect ratio). This is a quick fix, and we highly recommend [using go2rtc for live views](https://docs.frigate.video/configuration/live/) as it provides a superior experience with much more fluent and clear video.
```

The dimensions for the detection and recordings satisfy the divisible by 4 requirement.  1920 / 4 = 480 and 536  / 4 = 134
The camera gives me two choices for dimensions for the sub feed, neither are "standard" 16:9 which wouldn't make sense for this camera because it purposefully stitches two cameras into one:
<img width="342" alt="Screenshot 2024-10-17 at 8 29 27 PM" src="https://github.com/user-attachments/assets/051665b6-607a-4196-9c84-c869bab23737">

The dimensions for the main feed is 5120 x 1440  which cannot be changed.  5120 / 4 = 1280, and 1440 / 4 = 360

If I set the detection to use a 16:9 aspect ratio (1920 / 1080) the images displayed are severely distorted.

**Reply by NickM-27** [reply, marked as answer]
I have the same camera, it is 32:9 so you can use the sub stream and set the detect resolution as 1920x540

**Reply by zestysoft** [reply]
Ah!  Perfect, thanks for the fix.
